src/core/mac/mac.hpp

```diff
@@ -592,6 +592,12 @@ class Mac
     void ResetCounters(void);
 
     /**
+     * This method will tell us if the beacon resp has joinable set or not
+     *
+     */
```

Comment:
nit: I suggest adding either doxygen comment @returns, or @retval to document return value of a non-void method/function.